ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+### Short description
+
+I was trying to do [...]
+
+### Steps to reproduce
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+
+### Expected results
+
+
+### Actual results
+
+
+### On which platforms did you notice this:
```

Comment:
I would change the text to "In which browsers did you notice this?". "Platform" suggests operating system to me which is not relevant most of the time. I would also leave out the list of browsers - we're missing some (for example, "Chrome on Android" and "iOS Safari" are both mobile but very different targets) and if we mention specific versions, they are going to get out of date. The current version of  Safari is 9.x for example.